### PR TITLE
SysAid add get file

### PR DESCRIPTION
**Pull request by giocupelli84**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SysAid/Integrations/SysAid/SysAid.py [hunk 1]
```diff
@@ -916,6 +923,22 @@ def service_record_attach_file_command(client: Client, args: Dict[str, Any]) ->
...
+
+    response = client.service_record_get_file_request(sr_id, file_id)
+
+    if response.status_code == 200:
+        file_data = response.content
+        return_results(fileResult(file_name, file_data))
+
+        attachment_list = []
+        attachment_list.append({"file_name": file_name, "data": file_data})
+        return_results(str(attachment_list))
```
Comment (MosheEichler) [inline Packs/SysAid/Integrations/SysAid/SysAid.py:939]: Why is this required?

File: Packs/SysAid/Integrations/SysAid/SysAid.py [hunk 2]
```diff
@@ -916,6 +923,22 @@ def service_record_attach_file_command(client: Client, args: Dict[str, Any]) ->
...
 
 
+def service_record_get_file_command(client: Client, args: Dict[str, Any]):
+    sr_id = str(args.get('id'))
+    file_id = str(args.get('file_id'))
+    file_name = str(args.get('file_name'))
+
+    response = client.service_record_get_file_request(sr_id, file_id)
+
+    if response.status_code == 200:
```
Comment (MosheEichler) [inline Packs/SysAid/Integrations/SysAid/SysAid.py:933]: We don't want to return something if the response status code is not 200?

File: Packs/SysAid/Integrations/SysAid/README.md [hunk 3]
```diff
@@ -19553,6 +19553,66 @@ There is no context output for this command.
...
+| File.SHA512 | String | The SHA512 of the file. |
+| File.Name | String | The full name with extension of the file | 
+| File.SSDeep | String | The SSDeep of the file | 
+| File.EntryID | String | The entryId of the file | 
+| File.Info | String | The info of the file | 
+| File.Type | String | The type of the file | 
+| File.MD5 | String | The MD5 of the file | 
+| File.Extension | String | The extension of the file | 
+
+There is no context output for this command.
```
Comment (MosheEichler) [inline Packs/SysAid/Integrations/SysAid/README.md:19589]: It's a little bit confused if there is output for this command (lines 19573 and 19589)

**Review by MosheEichler — COMMENTED** [review]
Hi @giocupelli84,
Thank you for your contribution!
Good work :)

Please update the pack version in the pack [metadata](https://github.com/demisto/content/blob/master/Packs/SysAid/pack_metadata.json#L5) file.
Plesae see my other comments.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by MosheEichler** [comment]
Once you fix my coments we're ready for a demo.
Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

File: Packs/SysAid/Integrations/SysAid/SysAid.py (hunk 1 above)
Comment (giocupelli84) [reply Packs/SysAid/Integrations/SysAid/SysAid.py:939]: You are right, I added those lines later.
I added them because running the command "!sysaid-service-record-get-file file_id=FILEID file_name=FILENAME id=IDTICKET" the command returns the download of the file in the WarRoom and context, but if this command is run inside an automation nothing is downloaded.
I added these 3 lines so that I would receive a string in the automation that contains all the file_name and file_data and be able to run the "fileResult" command from within the automation as well.

File: Packs/SysAid/Integrations/SysAid/SysAid.py (hunk 2 above)
Comment (giocupelli84) [reply Packs/SysAid/Integrations/SysAid/SysAid.py:933]: I corrected the integration by adding an error message in the case of else

File: Packs/SysAid/Integrations/SysAid/README.md (hunk 3 above)
Comment (giocupelli84) [reply Packs/SysAid/Integrations/SysAid/README.md:19589]: The output of the "fileResult" comes out exactly that way, I have attached the example screenshot of a file downloaded with the command "!sysaid-service-record-get-file file_id=FILEID file_name=FILENAME id=IDTICKET"

Screenshot 2023-11-02 093719

**Review by MosheEichler — APPROVED** [review]

**Comment by MosheEichler** [comment]
Hey @giocupelli84
Please fill in the form https://forms.gle/XDfxU4E61ZwEESSMA
Thank you!

**Comment by giocupelli84** [comment]
Good morning I filled out the contribution registration form.
Thank you for approving my proposed changes.
Have a good day and enjoy your work.

**Comment by MosheEichler** [comment]
> Good morning I filled out the contribution registration form. Thank you for approving my proposed changes. Have a good day and enjoy your work.

Thank you !!

Would you be able to send a video or schedule a [demo](https://xsoar.pan.dev/docs/contributing/demo-prep) for these changes?
Thanks again :)

**Comment by giocupelli84** [comment]
I attach the requested video, (I have blacked out only small portions so as not to read privacy-protected automation names), the following tests are present:
- creation of a new ServiceRecord
- attaching an image to the created ServiceRecord
- deletion of the context
- get of only the attachments parameter of the ServiceRecord so as to get the data needed to get the attachment
- get of the attached file and opening the file.


https://github.com/demisto/content/assets/147415442/052b3ba4-b21e-44a3-9c68-b351e65ca47a
